assets/queries/k8s/shared_host_pid_namespace/query.rego

```diff
@@ -0,0 +1,28 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    specInfo := getSpecInfo(document)
```

Comment:
Add metadata information to get a more accurate searchKey

metadata := document.metadata